src/store/messages/utils.matrix.ts

```diff
@@ -48,7 +48,7 @@ export function* mapMessageSenders(messages, channelId) {
     }
   }
 
-  if (!matrixIds.length) {
+  if (matrixIds.length) {
```

Comment:
I can confirm that an idiot developer wrote this code 🙂

Reply:
😁